__tests__/index.test.js

```diff
@@ -0,0 +1,56 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index');
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('GET /api/topics', () => {
+	test('GET:200 return an array of topics of the correct format', () => {
+		return request(app)
+			.get('/api/topics')
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.topic.length).toBe(3);
+				body.topic.forEach((topic) => {
+					expect(topic).toMatchObject({
+						slug: expect.any(String),
+						description: expect.any(String),
+					});
+				});
+			});
+	});
+});
+
+describe('GET /api/invalid-path', () => {
+	test('GET:404 returns no content message', () => {
+		return request(app)
+			.get('/api/invalid-path')
+			.expect(404)
+			.then(({ body }) => {
+				expect(body.message).toBe('Not Found!');
```

Comment:
This message could be something like 'wrong path' as this error is not related to missing specific data.